docs/src/main/sphinx/installation/rpm.rst

```diff
@@ -0,0 +1,93 @@
+================
+RPM package
+================
+
+Users can install Trino using the RPM Package Manager (RPM) on some Linux
+distributions that support RPM.
+
+The RPM archive contains the application, all plugins, the necessary default
+configuration files, default setups, and integration with the operating system
+to start as a service.
+
+..  warning::
+
+  It is recommended to deploy Trino using the Helm or :doc:`Docker <containers>`
+  installation methods. While Trino supports RPM installation, it is only
```

Comment:
Reword to say that the RPM is available for use, but not tested. (not supported.. there is no such thing as support from an open source project)